packages/common/src/ui/components/navigation/Tabs/TabKeybindings.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
+
+interface TabKeybindingsProps<T> {
+  tabs: T[];
+  onAdd?: () => void;
+  setCurrentTab: React.Dispatch<React.SetStateAction<T>>;
+}
+
+export function TabKeybindings<T>({
+  tabs,
+  onAdd,
+  setCurrentTab,
+}: TabKeybindingsProps<T>) {
+  const keybindings = (e: KeyboardEvent) => {
+    if (e.ctrlKey) {
+      tabs.forEach((tab, n) => {
+        if (e.key === `${n + 1}`) {
+          e.preventDefault();
+          setCurrentTab(tab);
```

Comment:
Maybe add a comment with what the key bindings are and what they suppose to do (for slow people like me who take longer than needed to parse the code ;-))

Reply:
Good point - thanks. have done 😁